DOCKERFILE

```diff
@@ -0,0 +1,23 @@
+FROM python:3.10.6
```

Comment:
move it to separate pr - and let's call this file `Dockerfile` - so github will automatically apply correct highlighting